data/human/transport missions.txt

```diff
@@ -3085,12 +3091,13 @@ mission "Paradise Fortune 4"
 	on decline
 		payment 525894400
 		"reputation: Republic" <?= -1000
-	
+		log "People" "Diana Howl" `A teenager who stole half a billion credits from her family's wealth, she decided to start a new life outside the paradise planets and you decided to pocket the substantial fortune she left behind to avioid being hunted down.`
```

Comment:
duplicate  `decided`

Reply:
That's a deliberate word choice for emphasis in this case.

Reply:
Fair enough. I still feel like the sentence is a bit clunky, maybe you could replace the `and` with a while, similar to the other change?

Reply:
While made sense on the other one because you had to go and do the returning while she started her new life. In this case though the decision making is more instantaneous. I'll have a think.

Reply:
I'm on board with the `and -> while` change.